jerry-core/ecma/base/ecma-gc.c

```diff
@@ -1345,8 +1351,9 @@ ecma_gc_free_properties (ecma_object_t *object_p) /**< object */
           break;
         }
 #endif /* ENABLED (JERRY_ESNEXT) */
-#if ENABLED (JERRY_BUILTIN_WEAKMAP) || ENABLED (JERRY_BUILTIN_WEAKSET)
+#if ENABLED (JERRY_BUILTIN_WEAKMAP) || ENABLED (JERRY_BUILTIN_WEAKSET) || ENABLED (JERRY_BUILTIN_WEAKREF)
         case LIT_INTERNAL_MAGIC_STRING_WEAK_REFS:
+        case LIT_INTERNAL_MAGIC_STRING_CONTAINER_WEAK_REFS:
```

Comment:
Do we really need two internal properties for weak refs? Can't we store all the references in the same collection, and check the type of the referring objects? It should always be either a WeakRef or a WeakMap/WeakSet.